src/main/java/de/retest/recheck/persistence/bin/KryoPersistence.java

```diff
@@ -102,12 +103,33 @@ public void save( final URI identifier, final T element ) throws IOException {
 		}
 	}
 
-	@SuppressWarnings( "unchecked" )
+	/**
+	 * You need to maintain the stream and are responsible for closing it properly.
+	 */
```

Comment:
This comment is not true anymore

Reply:
Changes were lost during rebase. How annoying. Readded them.